webpack.config.js

```diff
@@ -53,9 +55,12 @@ module.exports = (env = {}) => {
     context: bundleEntryDir,
```

Comment:
Is this even helpful anymore? It seems like we're skipping it in favor of direct paths

Reply:
It is still being used for the un-vueified bundles, but we can remove in the future.